enterprise/permanentpen.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Permanent Pen Plugin
+title_nav: Permanent Pen
+description: Apply formats while typing
+keywords: permanent pen copy text format style
+controls: toolbar button, contextmenu, menu item
+---
+
+Permanent Pen is a premium TinyMCE plugin for applying a pre-defined format (color, typeface and type style, or font) to a desired section of content. The ability to store information for reuse saves the user time and ensures consistency across the content.
```

Comment:
"color, typeface and type style, or font" 
I half want this to be "and" instead of "or", but I'm not sure that would have the effect I want either. And also, we both know I'm biased on the terminology here.

"...a desired section of content."
Could we clarify this as _new_ content, or it being an as-you-type thing? This reads to me like the select and apply situation I messaged you about should work.